packages/commons/src/fixtures/tennis-club-membership-event.ts

```diff
@@ -1236,6 +1236,26 @@ export const tennisClubMembershipEvent = defineConfig({
           })
         }
       ]
+    },
+    {
+      type: ActionType.ARCHIVE,
+      label: {
+        id: 'v2.event.tennis-club-membership.action.archive.label',
+        defaultMessage: 'Archive',
+        description:
+          'This is shown as the action name anywhere the user can trigger the action from'
+      },
+      forms: [PRINT_CERTIFICATE_FORM]
+    },
+    {
+      type: ActionType.REJECT,
+      label: {
+        id: 'v2.event.tennis-club-membership.action.reject.label',
+        defaultMessage: 'Reject',
+        description:
+          'This is shown as the action name anywhere the user can trigger the action from'
+      },
+      forms: [PRINT_CERTIFICATE_FORM]
```

Comment:
Defining these here is a bit pointless but if I won't then I'll get an exception here:
https://github.com/opencrvs/opencrvs-core/blob/6c49bafdccb58d36c477737cd6d93d746ed32650/packages/commons/src/events/utils.ts#L214